packages/terra-form-select/translations/fr.json

```diff
@@ -15,6 +15,7 @@
   "Terra.form.select.dimmed": "Grisé.",
   "Terra.form.select.expanded": "Zone combinée développée",
   "Terra.form.select.collapsed": "Zone combinée réduite",
+  "Terra.form.select.notselected": "{text} Non sélectionné",
```

Comment:
what about other locales ..? I see translations missing for below locales 
en-CA
en-US
fi-FI
fr-FR
es-ES
es-US
nl-BE
pt-BR
sv-SE

Reply:
Updated